packages/@glimmer/component/test/ember/integration/components/glimmer-component-test.js

```diff
@@ -1,10 +1,12 @@
 import GlimmerComponent from '@glimmer/component';
-import { module, test } from 'qunit';
+import { module, test, skip } from 'qunit';
```

Comment:
We must not be linting something here, `skip` is unused (and that lint failure isn't failing the build) 🤔 

Not a giant issue, just isn't used any longer